<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UC_Clients</class>
 <widget class="QFrame" name="UC_Clients">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QFrame" name="Frame_Clients">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>860</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="Frame_Clients_Info">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1031</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_272">
     <property name="geometry">
      <rect>
       <x>865</x>
       <y>80</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>الهاتف الفرعى</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPhone2">
     <property name="geometry">
      <rect>
       <x>525</x>
       <y>70</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtName">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>10</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>ألاسم</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tbl_client">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>230</y>
       <width>1024</width>
       <height>562</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>الإسم</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>الهاتف الاساسى</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>الهاتف الفرعى</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>العنوان</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>البريد الالكترونى</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>من الملاك</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="txtPhone1">
     <property name="geometry">
      <rect>
       <x>525</x>
       <y>10</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add_client">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>190</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>إضافة</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chxOwners">
     <property name="geometry">
      <rect>
       <x>385</x>
       <y>140</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>من الملاك</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtAddress">
     <property name="geometry">
      <rect>
       <x>525</x>
       <y>130</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_remove_client">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>190</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>حذف</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>875</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>الهاتف الاساسى</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxOwners">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>130</y>
       <width>331</width>
       <height>39</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>البريد الإلكترونى</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtClienSearch">
     <property name="geometry">
      <rect>
       <x>675</x>
       <y>190</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-right:10%;</string>
     </property>
     <property name="placeholderText">
      <string>بحث سريع بالإسم</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEmail">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>70</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>875</x>
       <y>140</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="text">
      <string>العنوان</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_edit_client">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>simplified arabic</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>تعديل</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Frame_Clients_Reports">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1031</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clients_frame">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>simplified arabic</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>بيانات العملاء</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clients_archieve_frame">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>150</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>simplified arabic</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>أرشيف العملاء</string>
    </property>
   </widget>
   <zorder>Frame_Clients_Reports</zorder>
   <zorder>Frame_Clients_Info</zorder>
   <zorder>btn_clients_frame</zorder>
   <zorder>btn_clients_archieve_frame</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
